package.json

```
@@ -61,6 +62,7 @@
     "typescript": "^5"
   },
   "lint-staged": {
-    "*.js": "eslint --cache --fix"
+    "**/*.{cjs,js,jsx,ts,tsx}": "npx eslint",
```

Fant ut at du ikke trenger `npx`
```suggestion
    "**/*.{cjs,js,jsx,ts,tsx}": "eslint",
```